datastore.go

```diff
@@ -49,21 +54,34 @@ func NewDatastore(path string, opts *Options) (*datastore, error) {
 	}
 
 	return &datastore{
-		DB:   db,
-		path: path,
+		accessor: &accessor{ldb: db},
```

Comment:
This `accessor` is created with the LevelDB DB itself. The `accessor` on `NewTransaction()` is created with the transaction. I'm not married to the naming.